### ISSUE TITLE
[Issue]: not all clients receive websocket commands/messages

### ISSUE THREAD

**Issue by jumoog** [jellyfin/jellyfin#11755]
### Please describe your bug

I noticed that when I open Jellyfin Dashboard in one tab and start playing a video in another tab, the websocket url is always the same `/socket?api_key=<api_key>&deviceId=<deviceId>` for both tabs. When I send a message or command via the REST API, only the last connected websocket client receives the message. If I close the last open websocket connection, the old one gets all the messages again.

I think this is the reason why my plugin successfully sends messages and seek commands to skip the intro/credits and sometimes it does not work. In my opinion, the messages/commands should be sent to all affected websockets clients for the affected session.

### Reproduction Steps

- Open Jellyfin in two tabs and control the session via REST API. 
- Only the last connected client will receive commands and messages.
- Close the last opened tab and repeat sending messages/commands.
- Now the other tab receives everything.

### Jellyfin Version

10.9.0

### Environment

```markdown
- Virtualization: Docker
- Clients: Browser
- Browser: Firefox
```

### Jellyfin logs

```shell
Nothing of relevance is happening in the logs.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jumoog** [comment]
The problem is that you can watch multiple videos in the same session and you can no longer remotely control which video you want to control. This should only be an issue in the browser. I can't think of any other client that could do this. But I don't know how for example the webOS app behaves.

**Comment by jumoog** [comment]
fixed by #14494
